appveyor.yml

```diff
@@ -68,6 +62,13 @@ install:
   - echo extension=php_pdo_mysql.dll >> php.ini
   - echo extension=php_fileinfo.dll >> php.ini
 
+  # install composer
+  - IF NOT EXIST C:\tools\composer.phar (
+        cd C:\tools
+        && appveyor DownloadFile https://getcomposer.org/composer.phar
+    )
+  - php -dmemory_limit=-1 C:\tools\composer.phar self-update 
```

Comment:
Does self update revert from 2.0 to a 1.x release? Or why is that working? :thinking: